src/webview/html/visjsDebuggerPanel.html

```diff
@@ -21,11 +21,18 @@
     <script type="text/javascript">
       window.addEventListener('message', (event) => {
         const message = event.data; // The JSON data our extension sent
-
         if (message.command === 'updateVisjs') {
           const container = document.querySelector('#visjs-network');
           const network = new vis.Network(container, message.data, message.options);
         }
+        if (message.command === 'exportVisjs') {
+          const canvas = document.querySelector('#visjs-network canvas');
+          const img = canvas.toDataURL('image/png');
+          var downloadLink = document.createElement('a');
+          downloadLink.download = 'export.png';
+          downloadLink.href = img;
+          downloadLink.click();
```

Comment:
This is the first time I have seen this pattern without [the `a` element being added to and removed from the DOM](https://stackoverflow.com/questions/19327749/javascript-blob-filename-without-link).
However, it seems to be working.